app/frontend/src/components/MarkdownInput.tsx

```diff
@@ -34,6 +34,11 @@ const useStyles = makeStyles((theme) => ({
       "& a": {
         color: theme.palette.primary.main,
       },
+      "& img": {
+        width: "100%",
+        maxWidth: "400px",
+        height: "auto",
```

Comment:
Isn't the default height `auto` anyway? Does it do anything since it is not set to a fixed value?